api/types.go

```diff
@@ -648,6 +602,17 @@ func FormatParams(params map[string][]string) (map[string]interface{}, error) {
 				case reflect.Slice:
 					// TODO: only string slices are supported right now
 					out[key] = vals
+				case reflect.Pointer:
```

Comment:
I'm not sure if we need this – similar fields such as `stream` seem to work okay without it.

Reply:
Ah I see this is for parsing the options.